openedx/core/lib/api/tests/test_view_utils.py

```diff
@@ -0,0 +1,66 @@
+"""
+Tests for (some of) the view utilities.
+"""
+from django.conf.urls import url
+from django.test.utils import override_settings
+from rest_framework.response import Response
+from rest_framework.test import APITestCase
+from rest_framework.views import APIView
+
+from openedx.core.djangoapps.content.course_overviews.models import CourseOverview
+from xmodule.modulestore import ModuleStoreEnum
+from xmodule.modulestore.tests.django_utils import SharedModuleStoreTestCase
+from xmodule.modulestore.tests.factories import CourseFactory
+
+from ..view_utils import DeveloperErrorViewMixin, verify_course_exists
+
+
+class MockAPIView(DeveloperErrorViewMixin, APIView):
+    """
+    Mock API view for testing.
+    """
+
+    @verify_course_exists
```

Comment:
Nice. This is a pattern I didn't know we can do.  Sounds like a python study group talk or lightening talk

Reply:
Thanks -- I might mention it in demos today.